CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -1782,12 +1783,16 @@ if userGuiIcon3D and vr3dGuis then
 	end
 end
 
+local function lookMenuEnabledChanged()
+	if InputService.VREnabled then
+		TopbarPanel3D:SetVisible(lookMenuEnabled)
```

Comment:
If we enable the look menu we probably want to change the SetGUIInsetBounds back to 0, 0, 0, 0